### PR TITLE
BE-44 Implement text extraction method for text files

### PR DESCRIPTION
**Pull request by DeMensa123**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,4 +24,7 @@ To create all necessary tables in database, run:
 ## Code formatting
 Before committing, format code using `black` formater:
 
-```black .```
\ No newline at end of file
+```black .```
+
+## Text extraction
+To run textract package on Windows you need to add `c:\Program Files\Git\mingw64\bin\` to your PATH variables.
```
Comment (KToldy) [inline README.md:30]: How to install on Linux/Mac?  Check similar request here:
image

File: antiplag/nlp/text_preprocessing.py [hunk 2]
```diff
@@ -14,7 +14,20 @@
...
+import os
 
+def text_extraction(path_to_file):
+    file_name, file_extension = os.path.splitext(path_to_file)
+
+    if file_extension == '.srt' or file_extension == '.md':
+        file_extension = '.txt'
+        path_to_file_old = path_to_file
+        path_to_file = file_name + file_extension
+        os.rename(path_to_file_old, path_to_file)
```
Comment (KToldy) [inline antiplag/nlp/text_preprocessing.py:27]: Why do we change file names when we only want to extract the text?

**Review by KToldy — CHANGES_REQUESTED** [review]
Clarification requested

**Review by KToldy — APPROVED** [review]

File: antiplag/nlp/text_preprocessing.py (hunk 2 above)
Comment (MCFreddie777) [reply antiplag/nlp/text_preprocessing.py]: Seems like it's a little complicated - python [textract](https://github.com/deanmalmgren/textract) does not support `srt` and `markdown` files - therefore we move the content of the files into `txt` file which is later processed by `textract`.

I contacted Juro with this thing whether we should support this type of files or not.

**Review by OliverKanat — APPROVED** [review]

**Pull request merged**